### DISCUSSION TITLE
What browser versions does Storybook support?

### DISCUSSION THREAD

**Discussion by JamesIves** [storybookjs/storybook#29119] [Documentation] [answered]
I am attempting to asses some older browser support for my component library, however I'm unable to as Storybook doesn't seem to load as far back as Chrome 60+. We rely pretty heavily on Storybook for our testing/developer environments and have a sudden need to support these older versions.

My questions are:

1. Is there any documentation in regards to which browser versions the Storybook UI supports? I scanned the documentation site but couldn't find anything, other than a discussion thread that SB7 dropped IE11 support.
2. Is there any workarounds to get the UI to load so I can view the staged stories? Either through the usage of lots of polyfills, or a way to circumvent the UI altogether to just load the story iframes?

The nuclear option is to just load a separate environment with all of my components to validate these things, but that's likely going to be very time consuming considering the size.

**Comment by shilman** [comment, marked as answer]
@ndelangen  would know best but he's on vacation. Chrome 100 as I recall.

As a workaround, one thing you could try is to:
1. Bring up Storybook in a modern browser
2. Navigate to a story
3. Click the "eject" button in the upper right corner, which will open the story iframe in a new tab
4. Copy and paste the iframe URL into the old browser

Dunno whether it will help, but generally the Story iframe is under your control, whereas the outer iframe is owned by Storybook
